### PR TITLE
PLAT-16900: Updated neutral theme colors to resemble light theme.

### PR DESCRIPTION
**Pull request by Djspaceg**
Enyo-DCO-1.1-Signed-off-by: Blake Stephens <blake.stephens@lge.com>

### CODE REVIEW

File: css/moonstone-variables-dark.less [hunk 1]
```diff
@@ -1,7 +1,8 @@
 // Moonstone Dark Theme
 // ---------------------------------------
-@moon-dark-gray:   #4b4b4b;
-@moon-light-gray:  #ededed;
+@moon-dark-gray:   #4d4d4d;
+@moon-gray:        #a6a6a6;
```
Comment (otisnate) [inline css/moonstone-variables-dark.less]: Is ```@moon-gray```  just for reference of ```@moon-dark-gray``` and ```@moon-light-gray``` ?

Comment (Djspaceg) [inline css/moonstone-variables-dark.less]: We were missing a 'middle gray' color that we used in many places below.

Comment (otisnate) [inline css/moonstone-variables-dark.less]: Then, will you add 'middle gray' color to ```moonstone-variables-light.less``` file, too?

Comment (Djspaceg) [inline css/moonstone-variables-dark.less]: Sure. Why not.

**Pull request merged**